tests/instruments/smaract/test_scu_ascii_with_device.py

```
@@ -0,0 +1,248 @@
+    @pytest.mark.parametrize("channel", CHANNELS)
+    def test_stop(self, smaractascii, channel):
+        smaractascii.channels[channel].stop()
+
+# BAUDRATE TEST
+    @pytest.mark.parametrize("channel", CHANNELS)
+    @pytest.mark.parametrize("test_value, expected_outcome", [
+        (9600, 9600),
+        (1500, ValueError),
+    ])
+    def test_baudrate(self, smaractascii, test_value, expected_outcome, channel):
+        if expected_outcome == ValueError:
```

Ruff (0.15.12)

[error] Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)